src/pages/item-create/item-create.ts

```diff
@@ -41,18 +41,14 @@ export class ItemCreatePage {
   }
 
   getPicture() {
-    if (Camera['installed']()) {
-      Camera.getPicture({
-        targetWidth: 96,
-        targetHeight: 96
-      }).then((data) => {
-        this.form.patchValue({ 'profilePic': 'data:image/jpg;base64,' +  data });
-      }, (err) => {
-        alert('Unable to take photo');
-      })
-    } else {
+    this.camera.getPicture({
+      targetWidth: 96,
+      targetHeight: 96
```

Comment:
I suggest adding `destinationType: this.camera.DestinationType.DATA_URL` to the options, because the default is `FILE_URI` and then `this.form.patchValue` below doesn't work.